### ISSUE TITLE
really,how to build your solutions?

### ISSUE THREAD

**Issue by KnIfER** [KeenSoftwareHouse/SpaceEngineers#577]
branch1 failed with many ,many dll missings
——————————————————————————————————
branch2,branch3 compile success but run with many,many audio file missings.
Ignore them game do not crash.
But when create and enter a new world:
(debug): the game tell me world corrupt,take me back to main menu
(release):game throw an error and crash;in .log::::::::::::::::::::::::::::::::::::::::::::(I've forgotten what it said (0。0)
tiring……

UUUUUU  .log says" Exception occured:XXX same key XXX loaded" UUUUUU 

觞箍蟹聖愺睨馬駡

**Comment by KnIfER** [comment]
anyone?

**Comment by Krienas** [comment]
https://github.com/KeenSoftwareHouse/SpaceEngineers/issues/543 helped me in the past

**Comment by ESearcy** [comment]
I agree it's very frustrating, I've attempted getting this working on several occasions. 
make sure in steam you have (in the game properties) clicked the developer build/branch.
Then follow the instructions they give in the Proj. if that doesn't work, sorry bud

**Comment by Rseding91** [comment]
I don't understand how they (Keen software house) seemingly compile without any problems but then provide a broken repo here... ?

**Comment by spacebuilder2020** [comment]
This repo has not been updated since January.

**Comment by TDplai** [comment]
Sorry to be late to the party, but it also says that assets are not included (textures, sounds, etc.)

**Comment by Pingger** [comment]
The assets don't have anything to do with all those compile errors in their code.

**Comment by TDplai** [comment]
It does relate to Branch 2 and Branch 3 having missing audio files though

On Mon, Apr 9, 2018 at 10:13 PM, Pingger Shikkoken <notifications@github.com
